test/manager.js

```diff
@@ -1091,15 +1091,15 @@ describe('Manager', () => {
             expect(rendered).to.contain('<h1>default message</h1>');
         });
 
-        it('renders with an async global context function (no request)', async () => {
+        it('renders with an async global context function', async () => {
 
-            const asyncFn = () => Promise.resolve('from async');
+            const asyncFn = async () => await 'from async';
```

Comment:
The `await` here is required for the linter to not complain. We don't really need to await the result of this async func